yarn-error.log

```diff
@@ -0,0 +1,10044 @@
+Arguments: 
```

Comment:
This file was added unintentionally I presume?

Reply:
yes my bad